config/serverless.es.yml

```diff
@@ -61,6 +61,11 @@ xpack.ml.compatibleModuleType: 'search'
 
 data_visualizer.resultLinks.fileBeat.enabled: false
 
+# Enable "Other tools"
+xpack.canvas.enabled: true
```

Comment:
@TattdCodeMonkey are you sure you want to enable the Graph and Canvas plugins in Serverless? IIRC, this was never planned before and the PR description doesn't  explicitly call this out. 

/cc @lukeelmers

Reply:
cc @teresaalvarezsoler  Do you know more?

Not sure where this is coming from. Can we discuss?

thx @elmers for pinging.

Reply:
I'll verify with product, but they were explicitly included in the alignment of the navigation tasks.

Reply:
This came from @elastic/search-ux & @serenachou 

I believe we are trying to align with what we offer in stateful search solution navigation. But I'll let them elaborate further.

Reply:
Canvas is absolutely disabled in Serverless.  It's in maintenance mode and should not be enabled.

@ThomThomson @elastic/kibana-presentation

Reply:
Are both Canvas and Graph ins maintenance mode, or just Canvas?

Reply:
cc @elastic/kibana-visualizations for Graph.

Reply:
@linyaru Michael left a comment on the linked internal issue - the plan was that Canvas, Graph, and TSVB would be excluded from all solution menus in serverless.

Reply:
@linyaru for now I've removed Canvas & Graph from this PR